mirrord-config/src/lib.rs

```diff
@@ -44,6 +44,14 @@ pub struct LayerFileConfig {
     #[config(env = "MIRRORD_CONNECT_TCP")]
     pub connect_tcp: Option<String>,
 
+    /// Agent name that already exists that we can connect to.
+    #[config(env = "MIRRORD_CONNECT_AGENT")]
```

Comment:
Why is this a configuration value? When would we want users to be able to set it?

Reply:
It might prove useful for testing and maybe later scenarios. It's not for users though... it's just that the config is the relevant place (like connect_tcp)?